src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -463,6 +464,10 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
             ov::pass::ConvertQuantizeDequantize);
     }
 
+    CPU_REGISTER_PASS_COMMON(manager, ov::pass::EnableDecompressionConvertConstantFolding);
+    CPU_REGISTER_PASS_COMMON(manager, ov::pass::KeepConstAndDecompressionForMatMul);
+    CPU_REGISTER_PASS_COMMON(manager, ov::pass::ConstantFolding);
```

Comment:
We need to add comments why such behavior (enable and then disable decompression) is needed.

Reply:
added comment, @dmitry-gorokhov take a look please?